docs/operate-and-deploy/installation/upgrading.md

```diff
@@ -42,6 +42,30 @@ upgrading a cluster involves leaving the old cluster running on the old version,
 cluster on the new version, porting across your database schema, and finally thinking about your data.
 Read on for details.
 
+
+## (1) In-Place upgrade
```

Comment:
```suggestion
### In-Place upgrade
```
By changing the subheading level, this section is now a subsection of the "How to upgrade" section above, rather than at the same level.